dd-trace-php/Dockerfile_fpm

```diff
@@ -4,8 +4,9 @@ USER root
 WORKDIR /home/circleci
 
 RUN apt-get update && apt-get -y install vim less build-essential rpm rpmlint lintian\
-    && gem install fpm\
-    && (cd /usr/local/bundle/gems/fpm-1.10.2/; patch -p 1 < /tmp/fpm_apk_pax_header.patch )\
+    && gem install fpm -v 1.11.0
+ADD fpm_apk_pax_header.patch /tmp
```

Comment:
We should move this `ADD` to L5 before the `RUN`, that way we can keep this entire `RUN` command in a single command.

Unless the patch file is likely to change often enough and need to be reapplied?


(this is a nitpick, not a bit deal if you say no)

Reply:
I think the Steps following ADD are so small in a `diff` that it probably doesn't change a lot - performance wise.
Also since this image would be rebuilt on docker hub, I think the first part will be rebuild from scratch either way. 

So long story short, I think any approach will work equally well here :D
So lets make it a 1 liner :)